src/main/java/org/example/validator/AuthValidator.java

```diff
@@ -0,0 +1,34 @@
+package org.example.validator;
+
+import org.example.cli.Login;
+
+/**
+ * The type Auth validator.
+ */
+public class AuthValidator {
+  private final String regexPattern = "^(?=.{1,64}@)[A-Za-z0-9_-]+(\\.[A-Za-z0-9_-]+)*@"
+      + "[^-][A-Za-z0-9-]+(\\.[A-Za-z0-9-]+)*(\\.[A-Za-z]{2,})$";
+
+  /**
+   * Is valid email string.
+   *
+   * @param login the login
+   * @return the string
+   */
+  public String isValidLogin(Login login) {
+
+    if (login.getEmail().isEmpty()) {
+      return "Please enter a email.";
+    }
+
+    if (!login.getEmail().matches(regexPattern)) {
+      return "Email does not match expected pattern.";
+    }
+
+    if (login.getPassword().isEmpty()) {
```

Comment:
Will this handle if I entered some spaces?

What about password length? Must be between 8 and 64 characters etc